.gitignore

```diff
@@ -7,4 +7,6 @@
 /backup
 /user_privileges
 /config-dev.inc.php
+/config.inc.php
```

Comment:
I don't want to add this to coreBOS because we version config.inc.php on production sites. We fork the base project for each of our installs and version the config.inc.php file with the installs preferences.
We then override those preferences with the config-dev.inc.php file for development.